protocol/packages/swap/src/astroport.rs

```diff
@@ -282,15 +401,17 @@ mod test {
         ));
     }
 
+    // revert TODO report a cargo bug that 'test' cfg is not applied
+    #[cfg(feature = "testing")]
     #[test]
     fn resp() {
```

Comment:
please add a corresponding test(s) for the request part: a few different cases of build and parse